### ISSUE TITLE
Can't hide a sensitive view

### ISSUE THREAD

**Issue by somebody32** [smartlook/smartlook-mobile-issue-tracker#115]
**Platform**
iOS simulator, haven't tried Android

**Technology**
React Native

**Build instructions [optional]**
RN 0.66.2

**Smartlook SDK version**
1.2.2

**Summary**
I'm trying to occlude a sensitive view inside the App following the official docs and still seeing the content on the recordings.

Tried
1. with a hook
```tsx
const ref = useSmartlookSensitiveRef<View>(true);

...
<View ref={ref} ... />
```

in the console I have `👀Smartlook: [Line 357] 'setViewIsSensitive:' isSensitive: 1` but content is visible.

2. The same effect with directly calling `registerBlacklistedView` and passing the ref (have a different debug message in the console but nothing more)

**Comment by krystofcelba** [comment]
Hi,
Could you please send a snippet of code where you do the hiding of the sensitive content?

Thanks!

**Comment by somebody32** [comment]
hi @krystofcelba,

as I've stated in the post, I've tried `useSmartlookSensitiveRef` and `registerBlacklistedView` to no avail.

I'm trying to debug the issue more, but I don't have anything extra to add: I've tried to hide plain `View` and `Text` components. I've initially thought a View being Animated and positioned absolutely with transform have to do something with it, but later tried to occlude static Views but also failed. 

I think I'll try to generate a new rn app to try to pinpoint the issue
